securedrop_client/gui/widgets.py

```diff
@@ -1150,26 +1150,32 @@ def update(self):
                 self.paperclip.hide()
             self.star.update()
         except sqlalchemy.exc.InvalidRequestError as e:
-            logger.error(
-                "Could not update SourceWidget for source %s: %s",
-                self.source.uuid,
-                e
-            )
+            logger.error(f"Could not update SourceWidget for source {self.source_uuid}: {e}")
             raise
 
-    def set_snippet(self, source, uuid=None, content=None):
+    def set_snippet(self, source, uuid=None, content=None) -> None:
```

Comment:
nit: could you add a type for this `source` parameter?

Reply:
yep